extension/jaegerremotesampling/internal/http_test.go

```diff
@@ -72,7 +72,10 @@ func TestEndpointsAreWired(t *testing.T) {
 			resp.Body.Close()
 
 			body := string(samplingStrategiesBytes)
-			assert.Equal(t, `{"strategyType":"PROBABILISTIC"}`, body)
```

Comment:
Does the type enum field even matter any more?

See https://github.com/jaegertracing/jaeger-idl/blob/main/proto/api_v2/sampling.proto#L97

> Legacy field that was meant to indicate which one of the strategy fields
> below is present. This enum was not extended when per-operation strategy
> was introduced, because extending enum has backwards compatiblity issues.
> The recommended approach for consumers is to ignore this field and instead
> checks the other fields being not null (starting with operationSampling).
> For producers, it is recommended to set this field correctly for probabilistic
> and rate-limiting strategies, but if per-operation strategy is returned,
> the enum can be set to 0 (probabilistic).

Reply:
I think you are correct. Its no longer needed. I changed the test to verify a given `ProbabilisticSampling` field.